README.md

```diff
@@ -25,6 +25,19 @@ make distclean
 ```
 
 ## Usage
+ 
+- If the tile is valid, `vtvalidate` will return an empty string. 
+- If the tile is invalid, `vtvalidate` will return the string output from vtzero. 
+- `vtvalidate` will throw an error if there is unexpected behaviour, for example an invalid parameter value or a corrupt compressed buffer.
+
+#### Type of validation
+`vtvalidate` validates tile data against [vtzero](https://github.com/mapbox/vtzero):
+- Tile data consistent with the [Mapbox vector tile spec - Version 2](https://www.mapbox.com/vector-tiles/specification/). Tiles created via Mapbox vector tile spec Version 1 will be flagged invalid.
+- Read tile layer(s) and feature(s)
+- Decode properties
+- Decode geometries
```

Comment:
Does validate decode and confirm they are OGC Simple? Or just able to decode and not doing the extra leg lifting of OGC simple check?

Reply:
Fixed!